requirements.txt

```diff
@@ -0,0 +1,15 @@
+-e .
+
+pytest
+pytest-cov
+flake8
+autoflake
+pep8
+autopep8
+
+stored
+backports.tempfile
+tensorflow
+scipy
```

Comment:
Just a note, the packages listed here should be required in tests only -- if you have packages required to run the code, they should be in the `setup.py` instead

Reply:
So just test-related packages? 
Then should we remove `tensorflow` from the keras-trainer `requirements.txt`